karabo_data/geometry2.py

```diff
@@ -216,13 +221,15 @@ def from_quad_positions(cls, quad_pos, asic_gap=2, panel_gap=29):
             x_orient = quads_x_orientation[quad]
             y_orient = quads_y_orientation[quad]
             p_in_quad = p % 4
-            corner_y = quad_corner[1] - (p_in_quad * (128 + panel_gap))
+            corner_y = (quad_corner[1] * px_conversion)\
```

Comment:
Stupid question: should those values be rounded to int, since they're representing pixels

Reply:
They're representing pixel units, but we want to maintain sub-pixel accuracy for these positions. We round them for plotting (that's the 'snapped' geometry), but the more accurate positions can still be useful for CrystFEL and PyFAI.